### DISCUSSION TITLE
Suggestion: Show exported pdf file side by side with the md file

### DISCUSSION THREAD

**Discussion by merlinuwe** [l1xnan/obsidian-better-export-pdf#178] [Ideas]
The problem is, that one needs sometimes lots of cycles of editing the markdown file and generate the pdf file.

Would it be possible to generate the pdf file and automagically show it side by side with the markdown file?

![image](https://github.com/l1xnan/obsidian-better-export-pdf/assets/4138869/4427139a-54fa-4143-8b5b-ba8725b18682)

Then a shortcut to calll the pdf generation may be all one needs to do after editing the markdown file.

**Comment by l1xnan** [comment]
Unable to open files outside of Vault. If you want to open it, you need to generate the file inside the vault. You can currently open the previously generated PDF file side by side in Obsidian. Next time, when you generate it, overwrite this file to achieve a similar effect. It just adds an extra step of manually opening the PDF file. A better approach would be to spend some time studying the Obsidian API.
